### PR TITLE
Identify Purpose - update to align with SC text

### PR DESCRIPTION
**Pull request by alastc**

### CODE REVIEW

File: understanding/21/identify-purpose.html [hunk 1]
```diff
@@ -2,143 +2,78 @@
...
+			<p>This success criteria requires the author to add the context, propose and meaning  of symbols, regions, buttons, links and fields so that user agents knows what they do and can adapt them to make them understandable for the user. It is achieved by adding semantics or metadata that provides this context. It is similar to adding role information (as required by 4.2.1) but instead of providing information about what the UI component is (such as an image) it provides information about what the component represents (such as a link to the home page).</p>
 
-      ++todo add pictures++
-      <h3>Meeting the success criteria</h3>
-<p>Common ways to meet this success criteria include:</p>
-			<ul>
-<li>Adding context using ARIA AUI vocabulary - these techniques work on all the controls but is not yet a stable specification.
-Adding context using schema.org </li><li>
-Adding context using Microdata and the ARIA ePub vocabulary </li><li>
+			<p>Identifying regions of the page allows people to use  their </p>
```
Comment (DavidMacDonald) [inline understanding/21/identify-purpose.html]: orphan sentence?

File: understanding/21/identify-purpose.html [hunk 2]
```diff
@@ -2,143 +2,78 @@
...
+			  
+			<p>Products for people who are non-vocal often use symbols to help users communicate. These symbols are in fact peoples' language. Unfortunately, many of these symbols are both subject to copyright AND not interoperable. That means end-users can only use one device, and cannot use content,  apps or AT that has not been made by a single company.</p>
 
-Providing the purpose of a field using autocomplete - these techniques work on all the common fields. </li></ul>
+			<p>This success criteria enables symbols to be interoperable so that symbol users can understand different content that was not just made by one company.  When users' symbols are mapped to the same nodes then users agents can load the user understandable symbol. That means people could buy the symbols and use them across different devices or applications. (Note that the symbols would still be proprietary, but they could then be interoperable.)</p>
```
Comment (DavidMacDonald) [inline understanding/21/identify-purpose.html]: " users agents" should be "user agents"

File: understanding/21/identify-purpose.html [hunk 3]
```diff
@@ -2,143 +2,78 @@
...
-				<li> It enables for the ability to change levels of complexity (increase or decrease) - As skills improve or decrease over time or context. </li>
-				<li> It enables us to really meet user needs </li>
+				<li>Language</li>
+				<li>Memory</li>
+				<li>Focus and attention</li>
+				<li>Executive function and decision making.</li>
 			</ul>
-			<p>This helps people with many different cognitive disabilities including people with:</p>
+
+			<p>Meeting Success Criterion helps users who need extra support or a familiar interface, including the need for:</p>
```
Comment (DavidMacDonald) [inline understanding/21/identify-purpose.html]: Meeting <add>this </add> Success...

**Comment by alastc** [comment]
Thanks David, I've done those updates.

File: understanding/21/identify-purpose.html [hunk 4]
```diff
@@ -2,143 +2,78 @@
...
-			<p>For example, having familiar terms and symbols is key for users with a limited vocabulary to being able to use the web. However, what is familiar for some users may not be for other users, requiring them to learn new symbols. Personalization could include loading a set of symbols that is appropriate for the specific user, ensuring that all users find the icons simple and familiar.</p>
-
-			<p>Technology holds the promise of being extremely flexible. The design of many systems includes the expectation that users will be able to optimise their interaction experience according to their personal preferences or accessibility requirements (needs).</p>
-	
+			<p>The intent of this success criterion is to support personalization and preferences in order for more people to use the web, communicate and interact with society.</p>
+			<p>Having familiar terms and symbols is key for users with a limited vocabulary to being able to use the web. However, what is familiar for some users may not be for other users so programmatically associating user-interface components and icons enables people to load a set of symbols that is appropriate for them.</p>
```
Comment (awkawk) [inline understanding/21/identify-purpose.html]: Familiar terms and symbols are key...

**Pull request merged**